src/aap_eda/api/serializers/event_stream.py

```diff
@@ -89,8 +88,7 @@ def _get_extra_var_and_credential_ids(validated_data: dict) -> tuple[int, int]:
         validated_data, extra_vars, encrypt_vars, password
     )
 
-    extra_var = models.ExtraVar.objects.create(extra_var=yaml.dump(extra_vars))
-    return extra_var.id, credential_id
+    return extra_vars
```

Comment:
Minor suggestion: it's better to rename the function name to remove `credential_ids`, something like `_get_extra_var`, otherwise looks good to me.

Reply:
@hsong-rh Updated.